api/components/com_media/src/Model/AdaptersModel.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * @package     Joomla.API
+ * @subpackage  com_media
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Media\Api\Model;
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\MVC\Model\BaseModel;
+use Joomla\CMS\MVC\Model\ListModelInterface;
+use Joomla\CMS\Pagination\Pagination;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\Component\Media\Administrator\Event\MediaProviderEvent;
+use Joomla\Component\Media\Administrator\Provider\ProviderManager;
+
+/**
+ * Media web service model supporting lists of media adapters.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class AdaptersModel extends BaseModel implements ListModelInterface
+{
+	/**
+	 * A hacky way to enable the standard jsonapiView::displayList() to create a Pagination object,
+	 * since com_media's ApiModel does not support pagination as we know from regular ListModel derived models.
+	 *
+	 * @var int
+	 * @since  __DEPLOY_VERSION__
```

Comment:
These should be lined up 
```suggestion
	 * @var    int
	 * @since  __DEPLOY_VERSION__
```

Reply:
repeat for all the files with class properties like this